Actions/RunPipeline/RunPipeline.ps1

```diff
@@ -340,7 +343,36 @@ try {
         if ($repo."$_") { $runAlPipelineParams += @{ "$_" = $true } }
     }
 
-    Write-Host "Invoke Run-AlPipeline"
+    switch($buildMode){
+        'Clean' {
+            $preprocessorsymbols = $repo.cleanModePreprocessorSymbols
+
+            if (!$preprocessorsymbols) {
+                throw "No cleanModePreprocessorSymbols defined in settings.json for this project. Please add the preprocessor symbols to use when building in clean mode or disable CLEAN mode."
+            }
+
+            if (!$runAlPipelineParams.ContainsKey('preprocessorsymbols')) {
+                $runAlPipelineParams["preprocessorsymbols"] = @()
+            }
+
+            Write-Host "Adding Preprocessor symbols: $preprocessorsymbols"
+            $runAlPipelineParams["preprocessorsymbols"] += $preprocessorsymbols
+        }
+        'LCGTranslated' {
+            if (!$runAlPipelineParams.ContainsKey('features')) {
+                $runAlPipelineParams["features"] = @()
+            }
+            $runAlPipelineParams["features"] += "lcgtranslationfile"
+        }
+        'Translated' {

```

Comment:
Remove this.. When running with Translated (-> Default) buildmode we shouldn't add any parameters

Reply:
_Default_ build mode doesn't add anything extra